kubeflow/env.sh

```diff
@@ -1,9 +1,11 @@
 export KF_NAME=<kubeflow-cluster-name>
 export KF_PROJECT=<gcp-project-id>
+export KF_PROJECT_NUMBER=$(gcloud projects describe ${KF_PROJECT} --format='value(projectNumber)')
```

Comment:
Note, if sth goes wrong in $(), the script will not fail, so that's a silent failure and really hard to debug later.

Reply:
That is a good point, but I tried to fail the script inside $() on purpose, this will stop and output the error.

Do you mean the situation when the command in $() was wrong in producing the output, but the syntax is accurate? Should we advice user to provide project explicitly, instead of programming it for user?